experiments/test_non_cartesian.py

```diff
@@ -0,0 +1,141 @@
+import os
```

Comment:
To me this is more of a training script, so it should go in this directory.

Also be careful to skip lines between groups of imports.

Reply:
Oh wait actually it's already almost there in `nc_train` ! You just need to add the correct dataset.

Reply:
Yup, this was used by me for just quick debug, this can be deleted. Will do the same